client/my-sites/plugins/plugins-search-results-page/index.jsx

```diff
@@ -40,15 +35,12 @@ const PluginsSearchResultPage = ( {
 
 	const dispatch = useDispatch();
 
+	// Temporary until we decided to search or not within categories
+	const categoryName = null;
```

Comment:
I think your gut was right the first go round, lets just delete the dead code (category strings and all), its easy enough to bring it back and translation memory should pick them up quick enough too :crossed_fingers: